src/homepageExperience/components/FeedbackBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {ComponentSize, FlexBox, JustifyContent} from '@influxdata/clockface'
+import {ThumbsDown, ThumbsUp} from './HomepageIcons'
+
+type OwnProps = {
+  wizardEventName: string
+}
+export default class FeedbackBar extends React.Component<OwnProps> {
+  private handleThumbsUpClick = () => {
+    // handle event here
```

Comment:
anything keeping us from adding these events in now?

Reply:
added, and added some color feedback to the buttons.